### DISCUSSION TITLE
Voltage of 12V Batterie

### DISCUSSION THREAD

**Discussion by scrichab** [Hyundai-Kia-Connect/kia_uvo#676] [Ideas]
a lot of KIA/Hyundai Owner have huge problems with "loosing" 12V Battery of unknown reason
is the Voltage of the 12V Battery (car battery) available?  - so it would be helpful for logging
maybe it´s the better information instead of car battery level?

**Comment by DJBenson** [comment]
The 12v battery percentage is available via the integration;
<img width="579" alt="image" src="https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/1013909/fb6a66df-059e-4763-8482-c59ddb9cdf4c">

Don't rely on it for automation though as in all the times my Niro battery died, I never got an alert from HA (probably because the battery dies between data refreshes or during "blackout" times). You'd be better off with a BM2-type monitor if you're worried about your 12v battery.
